layouts/shortcodes/security-products/detection-rules-granular-access.md

```diff
@@ -0,0 +1,17 @@
+By default, all users have full access to the detection rules. To use granular access controls to limit the [roles][101] that may edit a single rule:
+
+1. Click the vertical three-dot menu for the rule and select **Permissions**.
+1. Click **Restrict Access**. The dialog box updates to show that members of your organization have **Viewer** access by default.
```

Comment:
Can you clarify what the "by default" part means here? Does this mean that all new users will, by default, only have view access to the rule going forward?

Reply:
Answered below.